wurst/event/DamageType.wurst

```diff
@@ -1,10 +1,11 @@
-// Use this package for finding the damage type in the context of a
-// DamageDetection handler. Typically this means differentiating between spells
-// and basic attacks.
+/** Use this package for finding the damage type in the context of a
+    DamageDetection handler. Typically this means differentiating between spells
+    and basic attacks. 
+    
+    It is deprecated tho, since there's DamageModify package. */
```

Comment:
Use proper english.

Also I don't think this is a strong enough warning. The key concern is that you definitely shouldn't use both this library and `DamageModify`, right?